src/Evolution/Executables/GeneralizedHarmonic/EvolveGhBinaryBlackHole.hpp

```diff
@@ -157,6 +157,14 @@
 #include "Utilities/Serialization/RegisterDerivedClassesWithCharm.hpp"
 #include "Utilities/TMPL.hpp"
 
+// Check if SpEC is linked and therefore we can load SpEC initial data
+#ifdef HAS_SPEC_EXPORTER
+#include "PointwiseFunctions/AnalyticData/GeneralRelativity/SpecInitialData.hpp"
+using SpecInitialData = gr::AnalyticData::SpecInitialData;
+#else
+using SpecInitialData = void;
```

Comment:
What do you think about using `Utilitias/NoSuchType.hpp` instead of `void`? A bit clearer maybe?

Reply:
I have a (very mild) preference for `void` because it's standard C++ and IMO quite clear

Reply:
I guess I find the opposite for the exact same reason. `void` can mean different things and in C++17 is (in some context) a valid type, while `NoSuchType` is meant to signify that some compile-time thing went wrong.  I think the confusing times is when you hit `void` in a `tmpl` context and then the question is "Is void good or bad?"

Reply:
Fine with me, changed to `NoSuchType`